<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractorForm</class>
 <widget class="QWidget" name="ExtractorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoJson Extractor</string>
  </property>
  <widget class="QGroupBox" name="LocationGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>[ 행정구역 쉐이프(SHP) 파일  ]</string>
   </property>
   <widget class="QLabel" name="SigFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>시군구</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SidoFilePathButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SidoFilePathEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SigFilePathButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLabel" name="SidoFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>시도(*)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SigFilePathEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="EmdFilePathButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmdFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>읍면동</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmdFilePathEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ResultGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>[ 결과물 저장 경로 ]</string>
   </property>
   <widget class="QPushButton" name="ResultFilePathButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ResultFilePathEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ResultFilePathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더(*)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>300</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HelpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>도움말</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
